spec/integration_spec.rb

```diff
@@ -9,12 +9,17 @@
 
   let(:actor_class) { Struct.new(:flipper_id) }
 
-  let(:admin_group) { flipper.group(:admins) }
-  let(:dev_group)   { flipper.group(:devs) }
+  let(:admin_group)  { flipper.group(:admins) }
+  let(:dev_group)    { flipper.group(:devs) }
+  let(:truthy_group) { flipper.group(:truthy) }
+  let(:falsey_group) { flipper.group(:falsey) }
```

Comment:
I think I was going to use them and then realized I could just use admin?. I'll remove them. Good catch.